cfgov/v1/util/util.py

```diff
@@ -15,14 +17,20 @@ def to_camel_case(snake_str):
     components = snake_str.split('_')
     return components[0] + "".join(x.title() for x in components[1:])
 
+
+def get_unique_id(prefix='', suffix=''):
+    index = hex(int(time()*10000000))[2:]
```

Comment:
So I think it depends on float point precision.  `time.time()` returns `1451584127.121017` at this moment.